pkg/nar/narinfo/types.go

```diff
@@ -88,6 +89,31 @@ func (n *NarInfo) String() string {
 	return buf.String()
 }
 
+// StoreDir derives in which prefix the store is located, based on the narinfo
+// StorePath field.
+func (n NarInfo) StoreDir() string {
+	i := strings.LastIndex(n.StorePath, "/")
+	return n.StorePath[:i]
+}
```

Comment:
We currently have a `nixpath.StoreDir` constant (which currently is only configurable by compiling with a different value). Parsing .narinfo files with other store prefixes is not supported. I'd probably use `nixpath.StoreDir` here as well - once/if we make this configurable, it'll be easier to spot the usages, and having this method might provide confusion, and cause people to assume we already support this.